src/ArcgisItemCard/ArcgisItemCard-styled.js

```diff
@@ -71,14 +87,20 @@ StyledItemCardImageWrap.defaultProps = { theme };
 
 const StyledCardItemTitle = styled(StyledCardTitle)`
   ${fontSize(1)};
-  margin-bottom: ${props => unitCalc(props.theme.baseline, 5, '/')};
+  margin-top: ${props => unitCalc(props.theme.baseline, 5, '/')};
```

Comment:
The margin looks off here, and when I compare against the card in the already deployed version. Can you make sure the styles stay consistent with the latest docs

Reply:
Would you recommend using padding here instead? Also, where would I need to go to find the styles in the already deployed version? Just within the other components?

Reply:
Nah I think margin is fine.

You could just visually compare agains the deployed app at https://calcite-react.netlify.app/ . If you need to see the code you can also look at the component files on github in the develop branch.